airflow-core/newsfragments/59688.improvement.rst

```diff
@@ -0,0 +1 @@
+Add ``proxies`` to ``DEFAULT_SENSITIVE_FIELDS`` in secrets_masker to treat proxy configurations as sensitive by default
```

Comment:
Can we also add "proxy" as sensitive name? I can easily imagine this one also being used in credentials with the same sensitivity

Reply:
Added both `proxy` and `proxies` as sensitive name.